src/core/thread/dua_manager.cpp

```diff
@@ -145,10 +144,10 @@ void DuaManager::HandleDomainPrefixUpdate(BackboneRouter::Leader::DomainPrefixSt
     }
 
     AddDomainUnicastAddress();
-#endif
 
 exit:
     return;
+#endif
```

Comment:
Should we use CI to make sure that such issue does not happen again? @librasungirl

Reply:
agree. see https://github.com/openthread/openthread/pull/5325#issuecomment-667551917